src/lib.rs

```diff
@@ -1,820 +1,651 @@
-//! Raw binding for RocksDB.
-//!
-//! This is just a thin binding to the RocksDB C API, mostly generated
-//! by rust-bindgen and lightly edited. This is intended to underpin a
```

Comment:
I regenerated it, then manually edited. Sorry, might have lost a few custom changes! Feel free to point them out.